test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperOutputTest.cs

```diff
@@ -207,6 +207,29 @@ public void GeneratePreContent_ReturnsNothingIfSelfClosing()
             Assert.Empty(output);
         }
 
+        [Theory]
+        [InlineData(true, null)]
+        [InlineData(true, "\t")]
+        [InlineData(false, null)]
+        [InlineData(false, "\t")]
+        public void GeneratePreContent_ReturnsPreContentIfTagNameIsEmpty(bool selfClosing, string tagName)
```

Comment:
- reverse parameter order
- nit: update test name to end with "IsWhitespace" or "IsNullOrWhitespace"